### PR TITLE
Dept 171

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ module:
   cookie_content_blocker: 0
   cookie_content_blocker_media: 0
   csp: 0
-  ctools: 0
```
Comment (johangant) [inline config/sync/core.extension.yml:26]: Ctools finally going!? Hooray :)

**Review by johangant — APPROVED** [review]
Just need the build/composer to resolve itself to a set of packages it can validate

File: config/sync/core.extension.yml (hunk 1 above)
Comment (omahm) [reply config/sync/core.extension.yml:26]: Yeah, pathauto was the only module with a dependency and that's now been sorted.

**Comment by johangant** [comment]
> I've removed a some package installs and the custom php config to get it passing. The new Cimg image includes libpng. I'm not sure what 'pcntl' and 'posix' was required for?

Those are for the platform.sh CLI tool

**Comment by omahm** [comment]
> > I've removed a some package installs and the custom php config to get it passing. The new Cimg image includes libpng. I'm not sure what 'pcntl' and 'posix' was required for?
> 
> Those are for the platform.sh CLI tool

Hmm that's going to be a problem

**Pull request merged**